packaging/build_packages_rpm.sh

```diff
@@ -37,16 +41,30 @@ el6_install(){
     n=$[$n+1]
     sleep 5
   done
+
+  import_gpg "6"
+}
+
+el7_install_import(){
+  n=0
+  while ! yum -y install https://rhel7.iuscommunity.org/ius-release.rpm; do
```

Comment:
This was a typo on my part, these should be https://centos7.iuscommunity.org/ius-release.rpm and https://centos6.iuscommunity.org/ius-release.rpm